index.html

```diff
@@ -126,6 +126,35 @@ <h3>Multi-Post Stories</h3>
             </ul>
             <button type="button" class="project-button">See Project</button>
         </section>
+        <section class="form-container">
+            <h3>CONTACT US</h3>
+            <P>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </P>
+            <form action="https://formspree.io/f/" method="POST">
+                <label for="name"></label>
+                <input type="text" id="name" name="username" placeholder="Text Input" required>
+                <label for="mail"></label>
+                <input type="email" id="mail" name="useremail" placeholder="jeremy@gmail.com"  maxlength="500" required>
+                <label for="msg"></label>
+                <textarea name="usermessage" id="msg" cols="30" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
+                <button class="button" type="submit">See Project</button>
+            </form>
+        </section>
```

Comment:
- [ ] Please fix the form action, it seems like you forget to pass the id after "https://formspree.io/f/". Kindly check the image below for more details
![form](https://user-images.githubusercontent.com/55635977/126396037-f618ef42-1505-4c9c-a4a5-372eb50b65ea.png)
- [ ] Change placeholder Text in input fields with your names